challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
+    
+    for(let i = 0; i < integers.length - 4; i++) {
+        integers[i] = "*";
+        cardNumber += integers[i]
+    }
+   
+    for(let i = integers.length - 4; i < integers.length; i++){
+        cardNumber += integers[i]
+    }
+    console.log(cardNumber)
+}
+
+
+hideCardNumber(123838383848)
+
+
+
+// second attempt 
+
+let encryptedCardNumber = "";
+
+
+function hideCardNumber(number) {
+   const numberString = number.toString()
+   const newNum = numberString.split('')
+
+
+  const hideNumber = "*"
+  const hidedenNum = hideNumber.repeat(newNum.length -4)
+  encryptedCardNumber = hidedenNum
+  
+  for(let i = newNum.length -4; i < newNum.length; i++){
+    encryptedCardNumber += newNum[i]
+  }
```

Comment:
You don't need to append the newNum to encryptedCardNumber. You can just concatenate the newNum and return it.

This is not that great, the first implementation was better. This is more redundant. Both are not great but the first one is slightly better.